.circleci/config.yml

```diff
@@ -52,6 +52,7 @@ jobs:
       - run:
           name: Install dependencies
           command: npm ci
+          no_output_timeout: 20m
```

Comment:
The default for this is 10 minutes. Our install on Circle CI takes ~13m30s, so this default would always cause our builds to fail due to a timeout. I have increased this to let the packages take the time they need because Circle's proxy connection to NPM is slow sometimes.